hack/install_kustomize.sh

```diff
@@ -1,10 +1,24 @@
 #!/bin/bash
 
-# Downloads the most recently released kustomize binary
-# to your current working directory.
+# If no argument is given -> Downloads the most recently released
+# kustomize binary to your current working directory.
+# (e.g. 'install_kustomize.sh')
+#
+# If an argument is given -> Downloads the specified version of the
+# kustomize binary to your current working directory.
+# (e.g. 'install_kustomize.sh 3.8.2')
 #
 # Fails if the file already exists.
 
+if [ -z "$1" ]
```

Comment:
Please stick to the formatting of other if/then statements.

Reply:
the formatting is now correct.